capitalize_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'base_decorator'
+
+class CapitalizeDecorator < Base_decorator
```

Comment:
- [ ] Also please make sure to fix this typo:

```suggestion
class CapitalizeDecorator < BaseDecorator
```